components/pipeline-service/staging/base/kustomization.yaml

```diff
@@ -8,17 +8,17 @@ commonAnnotations:
   argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
 
 resources:
-  - https://github.com/openshift-pipelines/pipeline-service.git/operator/gitops/argocd/pipeline-service?ref=e52f83e174efb8f308f6e32d1e0fc9f8eb9ed893
+  - https://github.com/openshift-pipelines/pipeline-service.git/operator/gitops/argocd/pipeline-service?ref=d9aba15332bf8e3e0f22b2d7e94747da491b20b2
   - pipelines-as-code-secret.yaml
   - ../../base/external-secrets
   - ../../base/testing
   - ../../base/rbac
 
 images:
   - name: quay.io/redhat-appstudio/tekton-results-api
-    newTag: bae7851ff584423503af324200f52cd28ca99116
+    newTag: d9aba15332bf8e3e0f22b2d7e94747da491b20b2
```

Comment:
well damn @enarha our automated update is changing our attempt to pin the image 

/hold

while I see how to disable or avoid that

Reply:
ok https://github.com/openshift-pipelines/pipeline-service/pull/997 works for me locally @enarha

Reply:
and its fixed i.e. it is staying at bae7851ff584423503af324200f52cd28ca99116 @enarha